src/main/java/uk/gov/pay/ledger/common/search/SearchParams.java

```diff
@@ -10,6 +10,10 @@
 
     public abstract Long getDisplaySize();
 
+    public boolean limitTotal() {
+        return false;
```

Comment:
Is this an oversight or on purpose?

Reply:
It is required and used here https://github.com/alphagov/pay-ledger/pull/871/files#diff-fe80f02b1b708a029ad3b2a5f520a889R52